text/0012-cva.md

```diff
@@ -0,0 +1,161 @@
+- that a validator may onboard as a result of attracting sufficient delegated stake weight - I.e. the validator's associated risk is based upon reputation rather than capital. 
+
+## Motivation
+
+[motivation]: #motivation
+
+It is imperative that the Casper network validator set can safely evolve over time.  It is inconceivable that the network does not support such a feature at genesis, i.e. it is a non-negotiable requirement of a minimal viable main-net.  Thus the CEP motivation is crystal clear, it is a necessary step towards a **demonstrably** healthy network.  Healthy from an operational, crypto-economic & reputational perspective.  
+
+A subset of the contextual use cases this CEP touches upon are:
+
+1.  The reputation of the network must be such that (new) validator's are willing to continue to assume non-trivial capital risk in order to participate;
+
+2.  The ratio of staked to non-staked token supply has game theoretic dynamics & hence real consequences;  
+
+3.  The inability of the token holder base to secure an ambient ROI (via inflationary rewards) results in a slow drift towards centralisation;
+
+## Guide-level explanation
+
+[guide-level-explanation]: #guide-level-explanation
+
+### 1. Establish a Casper Validator Association (CVA)
+
+Running a validator node entails significant risk.  Such risk revolves around the following elements:
+
+- Existential issues with the node software - e.g. a hitherto undiagnosed fault;
+- Operational issues with the node deployment - e.g. a concerted DDOS attack;
+- Jurisdictional issues associated with the business context - e.g. new compliance regulations.
+- Reputational issues within a social consensus context - e.g. cartel formation.
+
+A rational risk mitigation strategy dictates that validators collaborate.  Timely information sharing over well defined communication channels is the backbone of such collaboration. 
+Validators operating in isolation are disempowered as compared to collaborative ad-hoc groups of validators.  However ad-hoc groups tend to coalesce into political cliques focussing upon narrow tactical objectives at the expense of coherent strategic objectives.  This strategic incoherence is to the detriment of the network's long-term viability.
+
+An ad-hoc landscape is unfit for the purpose of nurturing and stewarding a successful crypto-economic network.  It is thus recommended that an association, hereby called the Casper Validator Association, is formally established.  Such an association will be endowed with financial capital and reputational credibility.  It seems logical to incorporate the association in either Singapore or Zug, Switzerland.  
+
+Association membership incurs both duties & responsibilities and all genesis validators will be required to be members of this association.  To focus the minds of association members the initial objectives will be to:
```

Comment:
This is starting to sound suspiciously like a joint stock company with an open ledger.